tests/common/mod.rs

```diff
@@ -87,6 +81,9 @@ pub fn for_each_block_from_disk(mut handler: impl FnMut(Block, InMemoryStorage))
                 })
                 .unwrap_or_default();
 
-        handler(block, InMemoryStorage::new(accounts, block_hashes));
+        handler(
+            block,
+            InMemoryStorage::new(accounts, bytecodes.clone(), block_hashes),
```

Comment:
Not important now, but this `clone` suggests that `InMemoryStorage` can also reference external data like `bytecodes` instead of owning it for better performance.